Expenso/Library/CoreData/MonthlyTransactionCD.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  MonthlyExpenseCD.swift
+//  Expenso
+//
+//  Created by Hendrik Steen on 31.05.22.
+//
+
+import Foundation
+import CoreData
+
+public class MonthlyTransactionCD: NSManagedObject, Identifiable {
```

Comment:
Instead of new model, we can use the existing model [ExpenseCD](https://github.com/comhendrik/Expenso-iOS/blob/e927912bfc64a7271aaca31f9a16e4eac6852786/Expenso/Library/CoreData/ExpenseCD.swift#L23) with `frequency` enum. Which can store values such as `{ onetime, monthly }`. Use this frequency val to query.

Also we don't need a separate button (Add income every month) to create recurring expense. Let's add a toggle which can update the frequency value.

![IMG_514CF50EB555-1](https://user-images.githubusercontent.com/26275107/173205936-900b962b-22db-40be-968b-adf94931c7c3.jpeg)

Reply:
Is there a possibility that this feature will be included in the app in the App Store? I ask myself this question because you mentioned that this is a different approach. However, I will start working on it.